Packs/PAN-OS/Integrations/Panorama/README.md

```diff
@@ -4687,3 +4687,558 @@ Installs specific content update file.
 >|JobID|Status|
 >|---|---|
 >| 30 | Pending |
+
+
+
+### pan-os-platform-get-arp-tables
+***
+Gets all arp tables from all firewalls in the topology.
+
+
+#### Base Command
+
+`pan-os-platform-get-arp-tables`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| device_filter_string | String to filter to only show specific hostnames or serial numbers. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| PANOS.ShowArp.Summary.hostid | string | ID of PAN-OS host | 
```

Comment:
Very nice work on the README! Make sure that when you update ```string``` into ```String``` that you update the README as well

Reply:
Done